velox/core/QueryConfig.h

```diff
@@ -170,14 +172,23 @@ class QueryConfig {
     return get<bool>(kExprEvalSimplified, false);
   }
 
+  std::optional<std::string> spillPath() const {
```

Comment:
Would you add a comment to explain what this is used for and what should be the format of the path? Do we support different storage adapters here or only local files? What kind of structure do we create within the path? Who is responsible for cleaning up the no longer needed files on crash, etc.?